contact.html

```diff
@@ -41,7 +42,7 @@ <h2 class="m-heading">Get in touch.</h2>
            <textarea name="message" id="message" placeholder="I was wondering about availability and rates. I need help with the following:"></textarea>
          </div>
          <div class="form-group">
-         <input type="submit" value="Send" class="btn-dark">
+         <input type="button" value="SUBMIT" class="btn-dark">
```

Comment:
I know the referenced site is using an input field for form submit action. Can you instead make it a `button` element or `a` element?